### PR TITLE
Changelog v240

### PR DESCRIPTION
**Pull request by LiveFireExercise**
expertcoderz

### CODE REVIEW

File: MainModule/Server/Shared/Changelog.lua [hunk 1]
```diff
@@ -3,6 +3,14 @@ return {
 	"*Drag edges to expand*";
 	"*Report bugs/issues on our GitHub repository (https://github.com/Epix-Incorporated/Adonis)*";
 	"";
+	"[Patch v239c 2023-04-09 20:40 UTC] @LiveFireExercise";
+	"Minor settings desc change";
+	"Added section headers";
+	"Disabled anti-speed by default";
+	"(Git/ccuser44) Re add better nuke (#1062)";
```
Comment (watameln) [inline MainModule/Server/Shared/Changelog.lua:10]: Re-add**

Comment (LiveFireExercise) [reply MainModule/Server/Shared/Changelog.lua:10]: shush

Comment (watameln) [reply MainModule/Server/Shared/Changelog.lua:10]: 🐈

**Comment by LiveFireExercise** [comment]
updated to v240 because it has a feature update

**Comment by LiveFireExercise** [comment]
✅ I give this a 10/10

**Comment by watameln** [comment]
cat out of cat

**Comment by LiveFireExercise** [comment]
"mrrp"
- you

**Comment by watameln** [comment]
Indeed

**Comment by WalkerOfBacon** [comment]
ok

**Comment by LiveFireExercise** [comment]
@Sceleratis 🥺

**Comment by LiveFireExercise** [comment]
thx scelly

**Pull request merged**